sycl/doc/UsersManual.md

```diff
@@ -446,6 +446,12 @@ and not recommended to use in production environment.
     When this option is used in conjunction with -fsycl, the driver will link
     against an alternate form of libsycl, libsycl-preview.
 
+**`-fno-spirv`**
```

Comment:
This description seems to be a bit misleading. It seems to suggest that users should use this option for all AOT target compilations. May be we can drop 'when targeting AOT devices'?